pylot/control/mpc/trajectory.py

```diff
@@ -0,0 +1,50 @@
+import numpy as np
+
+
+class Trajectory:
```

Comment:
We should probably move this to `utils.py` file.

Reply:
Done